root/static/scripts/edit/utility/guessFeat.js

```diff
@@ -18,211 +20,244 @@ import {
+  const isProbablyClassical = _.result(entity, 'isProbablyClassical');
 
-    var name = entity.name();
-    var match = extractFeatCredits(name, relatedArtists, isProbablyClassical, false);
+  const name = entity.name();
+  const match = extractFeatCredits(
+    name, relatedArtists, isProbablyClassical, false,
+  );
 
-    if (!match.name || !match.artistCredit.length) {
-        return;
-    }
+  if (!match.name || !match.artistCredit.length) {
+    return;
+  }
 
-    entity.name(match.name);
+  entity.name(match.name);
 
-    var artistCredit = entity.artistCredit().names.slice(0);
-    _.last(artistCredit).joinPhrase = match.joinPhrase;
-    _.last(match.artistCredit).joinPhrase = '';
+  const artistCredit = entity.artistCredit().names.slice(0);
+  _.last(artistCredit).joinPhrase = match.joinPhrase;
+  _.last(match.artistCredit).joinPhrase = '';
 
-    for (let name of match.artistCredit) {
-        delete name.similarity;
-    }
+  for (const name of match.artistCredit) {
+    delete name.similarity;
+  }
 
-    entity.artistCredit({
-        names: artistCredit.concat(match.artistCredit),
-    });
+  entity.artistCredit({
+    names: artistCredit.concat(match.artistCredit),
+  });
 
-    entity.artistCreditEditorInst && entity.artistCreditEditorInst.setState({
-        artistCredit: entity.artistCredit.peek(),
-    });
-};
+  entity.artistCreditEditorInst && entity.artistCreditEditorInst.setState({
+    artistCredit: entity.artistCredit.peek(),
+  });
+}
 
 // For use outside of the release editor.
 MB.Control.initGuessFeatButton = function (formName) {
-    var nameInput = document.getElementById('id-' + formName + '.name');
-
-    var augmentedEntity = Object.assign(
-        Object.create(MB.sourceRelationshipEditor.source),
-        {
-            // Emulate an observable that just reads/writes to the name input directly.
-            name: function () {
-                if (arguments.length) {
-                    nameInput.value = arguments[0];
-                } else {
-                    return nameInput.value;
-                }
-            },
-            // Confusingly, the artistCredit object used to generated hidden input
-            // fields is also different from MB.sourceRelationshipEditor.source's,
-            // so we have to replace this field too.
-            artistCredit: MB.sourceEntity.artistCredit
+  const nameInput = document.getElementById('id-' + formName + '.name');
+
+  const augmentedEntity = Object.assign(
+    Object.create(MB.sourceRelationshipEditor.source),
+    {
+      /*
+       * Emulate an observable that just reads/writes
+       * to the name input directly.
+       */
+      name: function () {
+        if (arguments.length) {
+          nameInput.value = arguments[0];
+        } else {
+          return nameInput.value;
         }
-    );
-
-    $(document).on('click', 'button.guessfeat.icon', function () {
-        guessFeat(augmentedEntity);
-    });
+      },
```

Comment:
> 244:13  error    Expected to return a value at the end of method 'name'  consistent-return

Reply:
Not sure what'd be a reasonable return for this one.